app/views/profile/index.html.slim

```diff
@@ -6,7 +6,10 @@
 
 - if current_user.active_profile
   - p = current_user.active_profile
-  h2.heading = t('headings.profile.profile_info')
+  h2.heading.mr1.inline-block
+    = t('headings.profile.profile_info')
+  span class="h5 blue sans-serif underline hint--top" aria-label=t('tooltips.profile_idv')
```

Comment:
I think this span also needs a `tabindex="0"` otherwise it won't ever get keyboard focus (since it's not actually a link).